Replace Google Analytics with Matomo

EcZX opened the pull request:
fix #45 
Replaced Google Analytics tracking code with Matomo Tracking Code

EcZX committed:
Update index.html

Replaced Google Analytics tracking code with Matomo Tracking Code

HaSistrunk commented:
Ok! I see a few things:

1. There is a Google Tag Manager (noscript) block of code that needs to be deleted in `index.html` lines 27-30 (but don't delete the `<body>` tag that comes right before it). We don't need to replace that with anything.
2. The Matomo script you used doesn't look like it's the one that corresponds to the Matchbox container in Matomo. Each site in Matomo Tag Manager has a unique container id, so it's best to copy the script directly from the "Install Code" script that Matomo provides for each site that you can find in Tag Manager.
3. It's best practice to give the pull request a descriptive title to explain what the changes do (similar to how we want commit messages to describe what that commit does). You can edit that - so something like "Replace Google Analytics with Matomo".

EcZX committed:
tag manger edits

replace correct tracking code for Matchbox and stripped out Google Analytics (nonscript)

HaSistrunk commented:
Nice! This looks good to me.

daremiyo reviewed (approved):
Looks good!
